github/issues.go

```diff
@@ -44,6 +44,7 @@ type Issue struct {
 	Repository       *Repository       `json:"repository,omitempty"`
 	Reactions        *Reactions        `json:"reactions,omitempty"`
 	Assignees        []*User           `json:"assignees,omitempty"`
+	ClosedBy         *User             `json:"closed_by,omitempty"`
```

Comment:
Yes, that is intentional - I didn't give access to named structs... I figured you could just use the accessors provided for that named struct, like:

`name := resp.User.GetName()`

(totally hypothetical - not looking at the code at the moment).  If you know that `resp` is not `nil`, then even if `User` is `nil` this should still work because `GetName` (or whatever field of the `User` struct you are interested in getting) can take `nil` as the receiver... if I'm making any sense.

I'm happy to change that if you think it would be useful, but I couldn't think of a scenario that would be useful.  (Note that I have not read this entire thread yet... and am just dropping by.)
Does that help any?  If not, I can investigate further.